.github/docker/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:bionic
+
+ENV HOME_BUILDBOT /var/lib/buildbot
+ENV PYTHON_PIP_VERSION 21.0.1
+# Set the desired version of Helm
+ENV DESIRED_VERSION v3.5.4
```

Comment:
Is this used ? Its the desired version of what ?

Reply:
Yes it's the Helm version, as the line 5 says it. 
(Shame on them for that naming)

Reply:
Ohh my bad, i focused on the not-so-clear variable name and missed the comment. If we dont have a choice lets keep it as is then.